tests/Integration/WhereConditions/WhereConditionsDirectiveTest.php

```diff
@@ -333,6 +335,338 @@ public function testAddsNestedAndOr(): void
         ]);
     }
 
+    public function testHasMixed(): void
+    {
+        factory(User::class, 9)->create()->each(function ($user) {
+            $user->posts()->saveMany(factory(Post::class, 2)->create());
+        });
+
+        factory(Comment::class, 1)->create([
+            'post_id' => 3,
```

Comment:
https://github.com/nuwave/lighthouse/blob/master/CONTRIBUTING.md#test-data-setup

Reply:
The ones where I make multiple creates, do I need to stick to that convention as well?

Reply:
Also If I create just a large batch of comments, and just assign specific ids to other comments. Does it goes against the convention to use the  ids directly?

Reply:
We rely heavily on static analysis. Using plain arrays with string keys totally circumvents that and should never be used.